taskcluster/ci/config.yml

```diff
@@ -142,9 +142,11 @@ scriptworker:
     scope-prefix: project:xpi:releng
     signing-format:
         by-xpi-type:
-            privileged: privileged_webextension
-            system: system_addon
+            hosted-privileged: privileged_webextension
             mozillaonline-privileged: privileged_webextension
+            normandy-privileged: privileged_webextension
+            system: system_addon
+            privileged: privileged_webextension
```

Comment:
We should remove `privileged` here; the 3-step migration is for xpi-template.

Reply:
2bebb23